### PR TITLE
Fist pull request

### PR DESCRIPTION
**Pull request by AtamaZack**
@tibetegya

### CODE REVIEW

**Comment by AtamaZack** [comment]
First pull request, please review and get back to me @tibetegya

File: api/ver1/views.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                error_dict['error {}'.format(error_dict['errors'])] = "Email already in use"
+        if error_dict['errors'] == 0:
+            hashed_pwd = generate_password_hash(
+                json_data['password'], method='sha256')
+            json_data['password'] = hashed_pwd
+            json_data['token'] = ""
+            del json_data['confirm password']
+            add_user(json_data)
+            return "Sign up successful!"
+        return error_dict
```
Comment (tibetegya) [inline api/ver1/views.py]: add a newline at the end of file

File: api/app.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from flask import Flask
+from flask_restful import Api
+from .ver1.views import Home, Signup
+
+APP = Flask(__name__)
```
Comment (tibetegya) [inline api/app.py]: make the app name lowercase

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
-# katale_stores
-This is a store management system.
+# Katale_stores
```
Comment (tibetegya) [inline README.md]: the name of the project changed please change it

**Comment by tibetegya** [comment]
Squash those commits into one commit

**Comment by tibetegya** [comment]
Add travis, code climate and coveralls badges

**Pull request closed without merging**